spec/src/main/asciidoc/core/injectionandresolution.asciidoc

```diff
@@ -668,14 +668,18 @@ The `Instance` interface provides a method for obtaining instances of beans with
 ----
 public interface Instance<T> extends Iterable<T>, Provider<T> {
 
-    public Instance<T> select(Annotation... qualifiers);
-    public <U extends T> Instance<U> select(Class<U> subtype, Annotation... qualifiers);
-    public <U extends T> Instance<U> select(TypeLiteral<U> subtype, Annotation... qualifiers);
+    Instance<T> select(Annotation... qualifiers);
+    <U extends T> Instance<U> select(Class<U> subtype, Annotation... qualifiers);
+    <U extends T> Instance<U> select(TypeLiteral<U> subtype, Annotation... qualifiers);
 
-    public boolean isUnsatisfied();
-    public boolean isAmbiguous();
+    Stream<T> stream();
```

Comment:
Please note the order.  I figure reading should be before the booleans and destroy.